ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -113,6 +113,46 @@ nnfw_get_accelerator (nnfw_pdata * pdata, const char *accelerators)
   }
 }
 
+/**
+ * @brief Parse accelerator and other option.
+ */
+static void
+nnfw_parse_option (const GstTensorFilterProperties * prop, nnfw_pdata * pdata)
```

Comment:
rename to `nnfw_parse_accl_custom`?

Reply:
Currently only accelerator, but I am not sure whether they would request another option for NNFW.
nnfw_parse_custom_option() ?

Reply:
nnfw_parse_custom_option() sounds good.

Reply:
updated